### PR TITLE
Ui redesign

### PR DESCRIPTION
**Pull request by Bozydarek**
Change some images, components and colors. Also I squashed some bugs in this PR.

### CODE REVIEW

File: app/src/utils/templatetags/time_utils.py [hunk 1]
```diff
@@ -15,3 +15,8 @@ def server_time():
 @register.filter
 def zoneformat(time, zonename):
     return format_in_zone(time, zonename, DEFAULT_TIME_FORMAT)
+
+
+@register.filter
+def isoformat(time):
+    return time.strftime("%Y-%m-%dT%H:%M:%SZ")
```
Comment (ref-humbold) [inline app/src/utils/templatetags/time_utils.py]: In ISO standard the letter 'Z' at the end of format string declares UTC time only. I'd rather first convert input time to UTC or specify offset instead.

File: app/src/conferences/models.py [hunk 2]
```diff
@@ -126,6 +126,10 @@ def end_date(self):
     def __str__(self):
         return 'Zosia {}'.format(self.start_date.year)
 
+    @property
+    def is_registration_started(self):
```
Comment (ref-humbold) [inline app/src/conferences/models.py]: Correctly should be 'is_registration_open' or 'has_registration_started'

File: app/src/utils/templatetags/time_utils.py (hunk 1 above)
Comment (Bozydarek) [reply app/src/utils/templatetags/time_utils.py]: JS can’t parse date if you don’t add this `Z` ...

Comment (ref-humbold) [reply app/src/utils/templatetags/time_utils.py]: Nope, you can omit it, but then it'll be treated as local time: https://www.ecma-international.org/ecma-262/#sec-date.parse

Comment (Bozydarek) [reply app/src/utils/templatetags/time_utils.py]: Is it ok right now?

Comment (ref-humbold) [reply app/src/utils/templatetags/time_utils.py]: OK

**Pull request merged**